File: clientapp/src/models/Account.ts
export class Account {
  id: number;
  userName: string;
  role: Role | null;

  constructor() {
    this.id = 0;
    this.userName = "";
    this.role = null;
  }
}
export default class Role {
  id: number;
  name: string;

  constructor() {
    this.id = 0;
    this.name = "";
  }
}

File: Models/Models/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MusicPlayer.Models
{
    public class Account
    {
        public int Id { get; set; }
        public string UserName { get; set; }
        public string Password { get; set; }
        public int RoleId { get; set; }
        public Role Role { get; set; }
    }
}
